cmd/agent.go

```diff
@@ -138,6 +138,12 @@ func init() {
 		os.Getenv("API_TOKEN"),
 		"Token used for authentication when API tokens are in use on the backend",
 	)
+	agentCmd.PersistentFlags().StringVar(
+		&agent.VenConn,
+		"venafi-connection",
+		"",
+		"VenafiConnection to be used",
+	)
```

Comment:
I think I'll also need to add a flag `--venafi-connection-namespace` along with `--venafi-connection`.

Reply:
It depends, if we want to support cross-namespace VenafiConnection references, we do.

Reply:
Got it. I've added the flag `--venafi-connection-namespace` to that effect.